request.go

```diff
@@ -31,25 +31,31 @@ type Config struct {
 	WaitHttp           time.Duration
 	HTTPRequestTimeout time.Duration
 
+	PubSubServer string
```

Comment:
It is quite confusing to user, what if: `RedisHost` since our lib is tightly couple with redis.